x-pack/legacy/plugins/code/index.ts

```diff
@@ -14,6 +14,8 @@ import { APP_TITLE } from './common/constants';
 import { codePlugin } from './server';
 import { PluginSetupContract } from '../../../plugins/code/server';
 
+export * from './shared';
```

Comment:
I believe this is what's causing the failure. You're importing client-side code here.

Reply:
Oh right, @rylnd you only need to export things from either public/index or server/index, not from the root index file. If it's browser code, you can export from public/index only.

Reply:
🤦‍♂, sorry folks, that's my bad. Thanks for the help!